lib/open_food_network/user_balance_calculator.rb

```diff
@@ -6,25 +6,14 @@ def initialize(email, distributor)
     end
 
     def balance
-      payment_total - completed_order_total
+      completed_orders.sum(&:outstanding_balance)
```

Comment:
I think this is correct but youare inverting the sign of the balance.
order.outstanding_balance is the positive valuee that buyer must pay.
this balance method here was "user balance".
order is 30 you have paid 10. order.outstand_balance = 20 this method -20